bcipy/main.py

```diff
@@ -178,7 +178,7 @@ def execute_task(
 def _clean_up_session(
         display: visual.Window,
         daq: ClientManager,
-        servers: List[LslDataServer] = None) -> bool:
+        servers: List[LslDataServer] = []) -> bool:
```

Comment:
Probably doesn't matter too much in this context, but this should be a listing error for a dangerous default. https://stackoverflow.com/questions/9526465/best-practice-for-setting-the-default-value-of-a-parameter-thats-supposed-to-be

Reply:
I remember reading that, and I think it has bitten us before (I vaguely recall Niklas mentioning it). I am trying to remember why I preferred this here... Probably to avoid the optional! 

After the multimodal merge, I need to fix some stuff, so I will revisit this before merging. Thanks for pointing out!